src/feature/controls_panel/manifest/CONTROLS_MANIFEST.ts

```diff
@@ -57,6 +58,11 @@ export const CTRL_TREE = {
               key: "NUMERIC_INPUT",
               child: null,
             },
+            {
+              title: "Array Numeric Input",
```

Comment:
Should it be `Numeric` or `Numberic`? Consistency should be maintained throughout all the files.

Reply:
that's a typo, thanks